Parsers/ASimDns/Product parsers/ASimDns.yaml

```diff
@@ -0,0 +1,23 @@
+Parser:
+  Title: Generic DnsEvents
+  Version: '0.1'
+  LastUpdated: May 11, 2021
+Product:
+  Name: Source agnostic
+Normalization:
+  Schema: Dns
```

Comment:
I didn't check it elsewhere, but DNS schema version is at 0.1.1

Reply:
The current version was released as 1.0.0, so I do not want to demote it. Maybe 1.1.0?